.circleci/config.yml

```diff
@@ -248,7 +248,8 @@ jobs:
     steps:
       - checkout
       - run: brew install swiftlint swiftformat
-      - run: swiftlint --strict
+      # Commenting out swiftlint for now because it started contradictiong swiftformat (https://github.com/mozilla/application-services/pull/3448)
```

Comment:
Instead of disabling swiftlint altogether, I suggest adding `opening_brace` under `disabled_rules` in `.swiftlint.yml`

Reply:
Done! That sounds better lol I had no idea that existed 😆